.pre-commit-config.yaml

```diff
@@ -99,6 +99,7 @@ repos:
             "--disable=c-extension-no-member",
             "--disable=import-error",
             "--disable=redefined-builtin",
-            "--disable=unused-wildcard-import"
+            "--disable=unused-wildcard-import",
+            "--disable=too-many-positional-arguments"
```

Comment:
It would be better to extend `pyproject.toml` configuration like:
```toml
[tool.pylint.design]
max-args = 11
max-positional-arguments = 9
max-locals = 30
max-branches = 15
max-returns = 8
```